docs/collecting-data/collecting-from-own-applications/python-tracker/tracking-specific-events/index.md

```diff
@@ -1,528 +1,282 @@
-  {
-    "movie_name": "Solaris",
-    "poster_country": "JP",
-    "poster_year": "1978-01-01"
-  }
-)
-```
-
-This is how to fire a page view event with both above contexts:
-
-```python
-t.track_page_view("http://www.films.com", "Homepage", context=[poster_context, geo_context])
-```
-
-**Important:** Even if only one custom context is being attached to an event, it still needs to be wrapped in an array.
-
-Note also that you should not pass in an empty array of contexts as this will fail validation. Instead of an empty array, you can pass in `None`.
-
-#### 2. Timestamp argument
-
-Each `track...()` method supports an optional timestamp as an argument. The timestamp should be in milliseconds since the Unix epoch, the same format as generated by `time.time() * 1000` .
-
-Generally, according to the Snowplow Tracker Protocol, every event tracked will be recorded with two timestamps:
-
-- the `dvce_created_tstamp`, which is the timestamp when the event was created
-
-- the `dvce_sent_tstamp`, which is the timestamp when the event was sent
-
-These are going to be used downstream, to calculate the `derived_tstamp` for the event, which takes also into account the collector timestamp, in order to best approximate the exact time the event occured.
-
-The optional timestamp argument is for the cases where you might want to set the event timestamp yourself. If this argument is not provided or set to None, then the Python Tracker will use the current time to be the `dvce_created_tstamp` for the event.
-
-Here is an example tracking a structured event and supplying the optional timestamp argument. We can explicitly supply `None` for the intervening arguments which are empty:
-
-```python
-t.track_struct_event("some cat", "save action", None, None, None, 1368725287000)
-```
-
-Alternatively, we can use the argument name:
-
-```python
-t.track_struct_event("some cat", "save action", tstamp=1368725287000)
-```
-
-**_\*\*Prior to v0.9.0_**
+Every tracked event payload has a unique event_id UUID string (`eid`). Other properties include the name_tracker (`namespace`) and `app_id` set when the Tracker was initialized. From version 1 onwards, `tracker.track()` returns the payload's `eid`.
 
-Before version 0.9.0 of the Python Tracker, providing a `snowplow_tracker.timestamp.TrueTimestamp` object as the timestamp argument will attach a true timestamp to the event, replacing the device timestamp. For example:
+Snowplow events have a defined structure and protocol that is identical regardless of the tracker used. A minimal payload - the raw event - is sent from the tracker to your collector. The raw event is [enriched](/enriching-your-data/what-is-enrichment/index.md) as it passes through your pipeline. By the time the event arrives in your data storage, depending which [enrichments](/enriching-your-data/available-enrichments/index.md) you have enabled, it will have gained different kinds of metadata, and have many more fields than it started with.
 
```

Comment:
This is nice but feels like we shouldn't explaining the whole pipeline in the docs of one of the trackers? Also use full links if you do keep it.